src/components/App/AppContainer.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import {AppComponent} from 'components/App/AppComponent';
```

Comment:
is linter configured to enforce no space or a single space between the brackets for the `import { }` statement?

Reply:
Not yet, which variant you prefer? I'll add correct rule.

Reply:
Any variant will do :) Would be great to follow some popular style like airbnb.

I can see you have already added the rule to have one space and updated the code. Well done!